src/tools/devtools/release-notes/release-notes-2.21.1-src.md

```diff
@@ -1,39 +1,69 @@
+  [#5062](https://github.com/flutter/devtools/pull/5062)
+* Improve the performance of taking a heap snapshot -
+  [#5134](https://github.com/flutter/devtools/pull/5134)
+* Retire broken import/export feature -
+  [#5135](https://github.com/flutter/devtools/pull/5135)
 
 ## Debugger updates
 
-* Added support for viewing profiler hits in the debugger script viewer - [#4831](https://github.com/flutter/devtools/pull/4831)
-* Added support for inspecting records - [#5084](https://github.com/flutter/devtools/pull/5084)
+* Added support for viewing profiler hits in
+  the debugger script viewer -
+  [#4831](https://github.com/flutter/devtools/pull/4831)
+* Added support for inspecting records -
+  [#5084](https://github.com/flutter/devtools/pull/5084)
 
 ## General updates
 
-* Fix several issues in syntax highlighting that would color variable names that contain reserved words incorrectly and leave `extends`/`implements` clauses uncolored for some classes - [#4948](https://github.com/flutter/devtools/pull/4948)
-* Fix an issue in Safari (browsers that do not support RegExp negative lookbehind) that prevented DevTools from loading - [#4938](https://github.com/flutter/devtools/pull/4938)
-* Fix an issue that would prevent DevTools connecting to the backend server that would disable some functionality - [#5016](https://github.com/flutter/devtools/pull/5016)
-* Add a link to the DevTools [CONTRIBUTING](https://github.com/flutter/devtools/blob/master/CONTRIBUTING.md) guide to the About menu, and fixed the Discord link - [#4926](https://github.com/flutter/devtools/pull/4926)
-* Fix conflicting colors in light theme - [5067](https://github.com/flutter/devtools/pull/5067)
+* Fix several issues in syntax highlighting that would
+  color variable names that contain reserved words incorrectly and
+  leave `extends`/`implements` clauses uncolored for some classes -
+  [#4948](https://github.com/flutter/devtools/pull/4948)
+* Fix an issue in Safari, and
+  other browsers that do not support RegExp negative lookbehind,
+  that prevented DevTools from loading -
+  [#4938](https://github.com/flutter/devtools/pull/4938)
+* Fix an issue that would prevent DevTools connecting to
+  the backend server that would disable some functionality -
+  [#5016](https://github.com/flutter/devtools/pull/5016)
+* Add a link to the DevTools
+  [contribution guide](https://github.com/flutter/devtools/blob/master/CONTRIBUTING.md)
+  to the About menu, and fixed the Discord link -
+  [#4926](https://github.com/flutter/devtools/pull/4926)
+* Fix conflicting colors in light theme -
+  [#5067](https://github.com/flutter/devtools/pull/5067)
 
 ## Full commit history
-More details about changes and fixes are available from the
-[DevTools git log.](https://github.com/flutter/devtools/commits/master).
 
+To find a complete list of changes since the previous release,
+check out
+[the diff on GitHub](https://github.com/flutter/devtools/compare/v2.20.0...v2.21.1).
```

Comment:
**question:** Again with the use of an include. That may work better here, too.

Reply:
At least currently we can't use constructors like the Jekyll variables or Liquid includes. I'll consider ways to do so in the future.